cert/selfsigned.go

```diff
@@ -272,10 +273,10 @@ func GenCertPair(org, certFile, keyFile string, tlsExtraIPs,
 	}
 
 	// Write cert and key files.
-	if err = ioutil.WriteFile(certFile, certBuf.Bytes(), 0644); err != nil {
+	if err = io.WriteFileToDisk(certFile, certBuf.Bytes(), 0644); err != nil {
 		return err
 	}
-	if err = ioutil.WriteFile(keyFile, keyBuf.Bytes(), 0600); err != nil {
+	if err = io.WriteFileToDisk(keyFile, keyBuf.Bytes(), 0600); err != nil {
 		os.Remove(certFile)
```

Comment:
I wrote the above comment and then looked one line below and happened to see a bug that the above suggestion would eliminate!